var/spack/repos/builtin/packages/gnutls/package.py

```diff
@@ -39,4 +39,8 @@ class Gnutls(AutotoolsPackage):
 
     version('3.3.9', 'ff61b77e39d09f1140ab5a9cf52c58b6')
 
-    depends_on("nettle")
+    # configure sez: Note that this version of gnutls doesn't support 
+    # nettle 3.0.
+    depends_on("nettle@:2.7.1")
```

Comment:
It says this:

```
[...]
checking for NETTLE... no
configure: error:
  ***
  *** Libnettle 2.7.1 was not found. Note that this version of gnutls doesn't support nettle 3.0.
```

I'm not an expert, but it seems that development's happening on the 3.x series.  2.7.1 as here: 2013-05-28 and everything since then has been on 3.x.

See here: https://ftp.gnu.org/gnu/nettle/

I don't know how to write spec so that it's less-than-but-not-equal-to 3.  I'm happy to change it if there's something cleaner....